ZipBuilder/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -182,6 +182,10 @@ struct ZipBuilder {
     // builds to just install a subset: `[.core, .analytics, .storage, .firestore]` for example.
     let subspecsToInstall = Subspec.allCases
 
+    // Remove CocoaPods cache so the build gets updates after a version is rebuilt during the
+    // release process.
+    CocoaPodUtils.cleanPodCache()
```

Comment:
How does this explain not finding GoogleToolboxForMac which isn't something that Firebase rebuilds?

Reply:
Even for non-rebuilds, sometimes random specs can't be found in the cache (even though they were *just* built. This would mean it's not in `pod cache list`, or maybe there's a bug in the parsing. Can you run `pod cache list` if you hit this error and see if `GoogleToolboxForMac` appears?